src/test/java/guitests/guihandles/PersonListPanelHandle.java

```diff
@@ -145,17 +222,114 @@ public String getFilterText() {
         return getTextFieldText(FILTER_FIELD_ID);
     }
 
+    /**
+     * Clicks the New button, which will open the edit dialog for creating new person.
+     * @return The EditPersonDialogHandle handler.
+     */
     public EditPersonDialogHandle clickNew() {
         guiRobot.clickOn(NEW_BUTTON_ID);
         guiRobot.sleep(500);
-        return new EditPersonDialogHandle(guiRobot, primaryStage);
+        return new EditPersonDialogHandle(guiRobot, primaryStage, EditPersonDialogHandle.ADD_TITLE);
+    }
+
+    /**
+     * Clicks the Edit button, which will open the edit dialog.
+     * @return The EditPersonDialogHandle handler.
+     */
+    public EditPersonDialogHandle clickEdit() {
+        guiRobot.clickOn(EDIT_BUTTON_ID);
+        guiRobot.sleep(500);
+        return new EditPersonDialogHandle(guiRobot, primaryStage, EditPersonDialogHandle.EDIT_TITLE);
     }
 
+    /**
+     * Drag and drop the person card.
+     * @param firstNameOfPersonToDrag The text which identify the card to be dragged.
+     * @param firstNameOfPersonToDropOn The text which identify the card to be dropped on top of.
+     */
     public void dragAndDrop(String firstNameOfPersonToDrag, String firstNameOfPersonToDropOn) {
         guiRobot.drag(firstNameOfPersonToDrag).dropTo(firstNameOfPersonToDropOn);
     }
 
     /**
+     * Drags the person cards outside of the listview.
+     * @param listOfPersonsToDrag The texts which identify the cards to be dragged.
+     */
+    public void dragOutsideList(List<String> listOfPersonsToDrag) {
+        double posY = this.getListView().localToScene(this.getListView().getBoundsInLocal()).getMaxY()
+                - 50;
```

Comment:
Improve line wrapping.
```java
double posY = this.getListView().localToScene(this.getListView().getBoundsInLocal()).getMaxY() 
              - 50;
```
Use explanatory variable for `this.getListView().localToScene(this.getListView().getBoundsInLocal()).getMaxY()`?